draft-ietf-tls-wkech.xml

```diff
@@ -68,59 +73,63 @@
   <middle>
     <section title="Introduction">
 
+	<t>
+	    The <xref target="RFC9460">Service Bindings (SVCB) in DNS RFC</xref>
+	    defines how to handle information needed to make connections
+        to services by using a new DNS record set (RRset) that defines
+        the HTTPS and SVCB resource record types.
+        Many origins will have an API that can be used to
```

Comment:
s/Many/Some/ ?? ignore if you want.

Reply:
I think Many is better - next sentence starts with some.